backend/typescript/models/user.model.ts

```diff
@@ -12,6 +13,9 @@ export default class User extends Model {
   @Column({ type: DataType.STRING })
   auth_id!: string;
 
-  @Column({ type: DataType.ENUM("User", "Admin") })
+  @Column({ type: DataType.ENUM("User", "Admin", "Volunteer", "Donor") })
   role!: Role;
+
+  @Column({ type: DataTypes.STRING })
+  phone_number!: string;
 }
```

Comment:
I believe email should also be added to the user model. Changes would need to be made in few other places as well to accommodate for this change.

Reply:
discussed, already referenced in `getUser()` service